module-system/sov-modules-api/Cargo.toml

```diff
@@ -68,5 +68,6 @@ native = [
     "macros",
     "sov-modules-macros/native",
     "sov-state/native",
+    "sov-rollup-interface/native",
```

Comment:
I can remove this, but in reality it is imported as "native" from `sov-state/native`, as `sov-db` imports rollup interface with native